src/components/NavBar/NavBar.vue

```diff
@@ -66,6 +71,12 @@ export default {
         this.$router.push({ name: "Login" });
       });
     },
+    redirect(symbol){
+      const regex = /^\S*$/;
+      if(symbol != "" || regex.test(symbol) == false){
```

Comment:
```suggestion
      const regex = /^\S+$/;
      if(regex.test(symbol)){
```

I'm not entirely sure what you are doing with this if statement it's complicated for no reason.
This regex will match any string that contains at least one character and no whitespace

Reply:
The purpose of this check is in the situation where we implement a proper search button so that people can't just search an empty search bar or an blankspaces (I should probably also add invisible characters too)